### DISCUSSION TITLE
How can I disable invite participants field from calendar module?

### DISCUSSION THREAD

**Discussion by davinhun** [YetiForceCompany/YetiForceCRM#16928] [How to] [answered]
Hello
I would like to disable or remove   invite participants field from the calendar module. 

I can't find the field here:
Software configuration/Standard modules/Edit fields.

How is it possible to hide it?

![image](https://user-images.githubusercontent.com/20862325/221678349-d4ce009b-52a3-490e-b773-c6baffe7f97c.png)

**Comment by rskrzypczak** [comment, marked as answer]
At the moment - no.
In the next version (>6.4) - yes (https://github.com/YetiForceCompany/YetiForceCRM/commit/eeb7733ded4b8b717d2f4e24556b532496a76a9b)

**Comment by vovpff** [comment]
You can hide this field by modification template files
